p/scripts/main.js

```diff
@@ -859,13 +859,17 @@ function init_stream(stream) {
 
 		el = ev.target.closest('.item.share > a[href="#"]');
 		if (el) {	//Print
-			const content = '<html><head><style>' +
-				'body { font-family: Serif; text-align: justify; }' +
-				'a { color: #000; text-decoration: none; }' +
-				'a:after { content: " [" attr(href) "]"}' +
```

Comment:
You're removing this here, but I'd definitely want the basic principle of it over to an `@media print` block in the common base stylesheet then.

That's serif instead of sans-serif, black links instead of blue (but I'm not sure I agree with a lack of underline), and the full URL in  `a::after`, although I'd probably just use regular parentheses instead of brackets.

Reply:
There is already a ```@media print``` block in the common base! At the moment, it's useless because we don't use it.

Reply:
I see, indeed. :-) It's used through the regular channels, just not through this share → print thing.